### ISSUE TITLE
[Errno 2] No such file or directory: after updating to windows 11

### ISSUE THREAD

**Issue by Rafeedy** [JarodMica/ai-voice-cloning#28]
Hi, i hope you can help me correct this issue. i am getting an errno 2 after i updated to windows 11. all was working prior to update. here is a video link to my error.
https://www.youtube.com/watch?v=xGAEB60rrf4
thanks

**Comment by Rafeedy** [comment]
![rvc-error2](https://github.com/JarodMica/ai-voice-cloning/assets/122225022/5807843f-708e-4beb-981c-d06648cf9268)

**Comment by JarodMica** [comment]
Is this a manual installation? If so, you can pull the repo.

Otherwise, you should be able to just create a folder named rvc_models inside of the models folder

**Comment by Rafeedy** [comment]
hi, thanks for responding, this was installation via your video instruction using the 7z method.
it was working great until i updated to windows 11.
i have already unzipped another new clean "ai-voice-cloning" in a different location and all is well working fine.
i just could not figure out why the older location stopped seeing the correct path, and the path was correct, i showed that in the video and pic i posted.
so in conclusion, the best way is to start fresh.

**Comment by ProtoPompAI** [comment]
<img width="1156" alt="screenshot err" src="https://github.com/JarodMica/ai-voice-cloning/assets/143017444/853a7fd9-b7c9-495f-bd8c-d06790b2f708">

To anybody else running into this issue, the culprit for me was checking `Run the outputted audio through RVC` and not selecting a model on the RVC drop down. This is under the `Show Experimental Settings` option under the generate tab. When you change these experimental settings, they persist after you reboot the program. The issue was fixed after I selected an RVC model on the dropdown menu.

**Comment by juntsukii** [comment]
Then after doing that it says "Something went wrong 'config'". Is it about the index file or how can i fix it?

**Comment by JarodMica** [comment]
@juntsukii Feel free to open a new issue if you still need help.  Closing this issue as original problem seems to be resolved.
